fe/fe-core/src/main/java/org/apache/doris/catalog/Catalog.java

```diff
@@ -6938,6 +6938,14 @@ public void replayBackendReplicasInfo(BackendReplicasInfo backendReplicasInfo) {
         List<BackendReplicasInfo.ReplicaReportInfo> replicaInfos = backendReplicasInfo.getReplicaReportInfos();
 
         for (BackendReplicasInfo.ReplicaReportInfo info : replicaInfos) {
+            if (tabletInvertedIndex.getTabletMeta(info.tabletId) == null) {
```

Comment:
can we invoke check replica or meta exist only once? it seems duplicate with following check